root/report/AnnotationsEvents.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {ANNOTATION_REPORT_TEXT} from './constants';
+import EventList from './components/EventList';
+import ReportLayout from './components/ReportLayout';
+import useAnnotationColumns from './hooks/useAnnotationColumns';
+import type {ReportEventAnnotationT, ReportDataT} from './types';
```

Comment:
Nit-picking: It should be alphabetically ordered (just like it already is in other `root/report/Annotations*.js`):
```suggestion
import type {ReportDataT, ReportEventAnnotationT} from './types';
```